### PR TITLE
added division

### PR DESCRIPTION
**Pull request by nshevelyn**
division has been added

### CODE REVIEW

File: date.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+console.log(10 / 3
+);
```
Comment (nshevelyn) [inline date.ts:2]: fghjk

**Comment by nshevelyn** [comment]
fgh

File: date.ts (hunk 1 above)
Comment (nnikolaienko) [inline date.ts:2]: ok

**Review by nnikolaienko — APPROVED** [review]

**Comment by nnikolaienko** [comment]
ok

**Pull request closed without merging**